content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -15,6 +15,48 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <!-- more -->
 
+## Deferred Rendering
+
+<div class="release-feature-authors">authors: @DGriffin91</div>
+
+The two most popular "rendering styles" are:
+
+* **Forward Rendering**: do all material/lighting calculations in a single render pass
+  * **Pros**: Simpler to work with. Works on / performs better on more hardware. Supports MSAA. Handles transparency nicely.
+  * **Cons**: Lighting is more expensive / fewer lights supported in a scene, some rendering effects are impossible (or harder) without a prepass
+* **Deferred Rendering**: do one or more pre-passes that collect relevant information about a scene, then do material/lighting calculations in _screen space_ in a final pass after that.
+  * **Pros**: Enables some rendering effects that are not possible in forward rendering. This is especially important for GI techniques. Can support more lights in a scene
+  * **Cons**: More complicated to work with. Requires doing prepasses, which can be more expensive than an equivalent forward renderer in some situations (although the reverse can also be true), doesn't support MSAA, transparency is harder / less straightforward.
+
+Bevy's renderer has historically been a "forward renderer". More specifically, it is a [Clustered Forward / Forward+](/news/bevy-0-7/#unlimited-point-lights) renderer, which means we break the view frustum up into clusters and assign lights to those clusters, allowing us to render many more lights than a traditional forward renderer.
+
+However, as Bevy has grown, it has slowly moved into "hybrid renderer" territory. In previous releases, we added a [Depth and Normal Prepass](/news/bevy-0-10/#depth-and-normal-prepass) to enable [TAA](/news/bevy-0-11/#temporal-anti-aliasing), [SSAO](/news/bevy-0-11/#screen-space-ambient-occlusion), and [Alpha Texture Shadow Maps](/news/bevy-0-10/#shadow-mapping-using-prepass-shaders). We also added a Motion Vector Prepass to enable TAA.
+
+In **Bevy 0.12** we added support for Deferred Rendering (building on the existing prepass work). Each material can choose whether it will go through the forward or deferred path, and this can be configured per-material-instance.
+
+For the PBR [`StandardMaterial`], the deferred prepass packs PBR information into the Gbuffer, which looks like this:
```

Comment:
I'd add a note that this is completely optional and the `StandardMaterial` still uses forward by default right now.

Reply:
Done!